packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -255,9 +299,27 @@ export default {
       this.validationService.checkFormInput("country");
       this.validationService.checkEmailInput("email");
       this.validationService.checkFormInput("state");
+      this.validationService.checkFormCheckboxInput("consentAgree");
 
       if (this.validationService.isValidForm()) {
-       // make api call
+        // make api call
+      }
+    },
+
+    showConsentSection(country) {
+      // Show consent section while selecting country.
+      this.displayConsent = true;
```

Comment:
What is the purpose of this variable? It seems like it only gets set to `true` but never gets toggled back anywhere else?

Reply:
as soon as user will select any country consent block becomes visible and it will be visible for all countries. At the same time, opt checkbox will be visible only for gdpr countries, you can check the current implementation of this part on prod version

Reply:
Ah, okay, thank you for clarifying.